ibis-server/resources/function_list/mysql.csv

```diff
@@ -1,4 +1,141 @@
 function_type,name,return_type,param_names,param_types,description
+scalar,tan,,,,Returns the tangent of a number.
```

Comment:
Could you add the return type for them? If we don't give a return type, it will use `Utf8` as the return type by default. It's not good for the type check or type coercion thing.
https://github.com/Canner/wren-engine/blob/1f220aa67e94331c244a21afd00fe5dc4f4a8edc/wren-core/core/src/mdl/mod.rs#L504

You can refer to the definition to choose a suitable type for them.
https://github.com/Canner/wren-engine/blob/b08706dc6e002fc0d2426e62737fa6c08c2fe07a/wren-core/core/src/mdl/function.rs#L95

To understand the usage, you can refer the function list of BigQuery or the test cases.